Source/animationinfos.h

```diff
@@ -17,35 +17,35 @@ class AnimationInfos {
 	/*
 	* @brief Pointer to Animation Data
 	*/
-	uint8_t *_pAnimData;
+	uint8_t *pData;
 	/*
 	* @brief Tick length of each frame in the current animation
 	*/
-	int _pAnimDelay;
+	int DelayLen;
 	/*
-	* @brief Increases by one each game tick, counting how close we are to _pAnimDelay
+	* @brief Increases by one each game tick, counting how close we are to DelayLen
 	*/
-	int _pAnimCnt;
+	int DelayCounter;
 	/*
 	* @brief Number of frames in current animation
 	*/
-	int _pAnimLen;
+	int FrameLen;
 	/*
 	* @brief Current frame of animation
 	*/
-	int _pAnimFrame;
+	int CurrentFrame;
 	/*
 	* @brief Specifies how many animations-fractions are displayed between two gameticks. this can be > 0, if animations are skipped or < 0 if the same animation is shown in multiple times (delay specified).
 	*/
-	float _pAnimGameTickModifier;
+	float GameTickModifier;
 	/*
 	* @brief Number of GameTicks after the current animation sequence started
 	*/
-	int _pAnimGameTicksSinceSequenceStarted;
+	int GameTicksSinceSequenceStarted;
 	/*
 	* @brief Animation Frames that will be adjusted for the skipped Frames/GameTicks
 	*/
-	int _pAnimRelevantAnimationFramesForDistributing;
+	int RelevantFramesForDistributing;
```

Comment:
The member doesn't specify how many frames are to distribute.
It specifies which Animation Frames are to be adjust (normal shortened) to compensate the skipped Frames.

Would something like `FramesToModifieForSkippedFrames` or `FramesToAdjustForSkippedFrames` help?
See also [animationinfo.cpp#66](https://github.com/diasurgical/devilutionX/pull/1695/files#diff-7cb2d4eccfa51b37e493533222462f44601d98dddc7dde5bf784be0edef6577aR63)

Reply:
> It specifies which Animation Frames are to be adjust (normal shortened) to compensate the skipped Frames.

But that sounds like a list to me. How is it a single int then? If it specifies "which frames", it would need to be a list of frames no?

Reply:
It implies Frame 1 to this specified frame. With that we have a list. 😉 
In theory it could also be frame 4 to 6 or something like this.
But in Diablo I don't think we ever need this. So I didn't complicate the logic.